From e19ba068bf6634d630e85113316ee1cb087a7f42 Mon Sep 17 00:00:00 2001
From: Michal Schmidt <mschmidt@redhat.com>
Date: Thu, 25 Sep 2014 09:37:42 -0400
Subject: [redhat] kernel.spec: bump required linux-firmware version

Message-id: <1411637862-3164-1-git-send-email-mschmidt@redhat.com>
Patchwork-id: 96715
O-Subject: [RHEL7.1 PATCH] redhat: bump required linux-firmware version in kernel.spec
Bugzilla: 1146120
RH-Acked-by: Ivan Vecera <ivecera@redhat.com>
RH-Acked-by: Nikolay Aleksandrov <nikolay@redhat.com>
RH-Acked-by: Jarod Wilson <jarod@redhat.com>

BZ: https://bugzilla.redhat.com/show_bug.cgi?id=1146120

Bump the required linux-firmware version to get the latest build
(linux-firmware-20140911-0.1.git365e80c.el7).
Several drivers got updated for RHEL 7.1 and they now require newer
firmware files than are present in linux-firmware from 7.0 (e.g. bnx2x).
Without the updated versioned dependency users may want to try running
a RHEL 7.1 kernel with 7.0 packages and be surprised when drivers fail
to find their firmware.

Upstream:
N/A. This is a Red Hat packaging issue.

Brew:
https://brewweb.devel.redhat.com/taskinfo?taskID=8010213

Testing:
I checked that yum now refuses to update the kernel alone. The update
proceeds if the new linux-firmware is also in the transaction.

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kernel.spec.template b/redhat/kernel.spec.template
index 3478650..1bb4e4c 100644
--- a/redhat/kernel.spec.template
+++ b/redhat/kernel.spec.template
@@ -255,7 +255,7 @@ Provides: kernel-modeset = 1\
 Provides: kernel-uname-r = %{KVRA}%{?1:.%{1}}\
 Requires(pre): %{kernel_prereq}\
 Requires(pre): %{initrd_prereq}\
-Requires(pre): linux-firmware >= 20100806-2\
+Requires(pre): linux-firmware >= 20140911\
 Requires(post): %{_sbindir}/new-kernel-pkg\
 Requires(preun): %{_sbindir}/new-kernel-pkg\
 Conflicts: %{kernel_dot_org_conflicts}\
-- 
1.7.1